AdyenCard/Components/Card/CardComponent.swift

```diff
@@ -140,7 +140,13 @@ public class CardComponent: PresentableComponent,
         
         cardViewController.stopLoading()
     }
-    
+
+    public func submit() {
```

Comment:
If this is the public function that we will expose to merchants to trigger on their button taps, let's add documentation and maybe a more fitting name.

Reply:
Yes, I'm planning to do the submit logic for all components in a different PR that will be mostly about it